app/models/translation.rb

```diff
@@ -8,40 +8,32 @@ class Translation < ApplicationRecord
   include NotifierConfig
 
   def self.translate(text)
-    Rails.cache.fetch("translations/#{text}", expires_in: 15.minutes) do
```

Comment:
This old implementation seems like it could cause a performance problem. My local db has ~1000 translation records so that's 1000 write operations to redis. Probably gets slower if multiple nodes miss at the same time and are all writing simultaneously